narwhal/source_code_sharing/update_liferay_dxp_source_code_of_hotfixes.sh

```diff
@@ -38,7 +43,7 @@ function check_ignore_zip_file {
 
 	local file_url="${zip_directory_url}/${hotfix_zip_file}"
 
-	if [[ "x${IGNORE_ZIP_FILES}" =~ x*${hotfix_zip_file}* ]]
+	if echo "${IGNORE_ZIP_FILES}" | grep -q "${hotfix_zip_file}"
```

Comment:
I just realized, this snippet should not be handled this way anymore.

process_argument_ignore_zip_files() processes the original file and creates the $IGNORE_ZIP_FILES variable which is not needed anymore. It would be better to grep that file directly